src/garage/np/algos/base.py

```diff
@@ -25,4 +28,34 @@ def train(self, runner):
             The average return in last epoch cycle or None.
 
         """
-        pass
+
+    def evaluate_performance(self, batch):
+        # pylint: disable=no-self-use
+        r"""Evaluate the performance of the algorithm.
+
+        Args:
+            batch (dict): Evaluation trajectories, representing
+                the best current performance of the algorithm, with key
+                * itr (int): Iteration number.
+                * num_trajs (int): Number of trajectories.
+                * episode_reward_mean (float): Average episode reward.
+                * average_discounted_return (float): Average discounted return.
+                * undiscounted_returns (numpy.ndarray): A numpy array of shape
+                    :math:`(N \bullet [T])` containing the undiscounted returns
+                    for all time steps in this batch.
+
+        """
+        tabular.record('Evaluation/Iteration', batch['itr'])
+        tabular.record('Evaluation/NumTrajs', batch['num_trajs'])
+        tabular.record('Evaluation/EpisodeRewardMean',
```

Comment:
i think you need to update several scripts in test/benchmarks with this new name

Reply:
this should be calculated here, from the batch itself